regression-test/suites/fault_injection_p0/cloud/test_cloud_mow_delete_bitmap_lock_case.groovy

```diff
@@ -0,0 +1,784 @@
+                file "test_stream_load3.csv"
+
+                time 10000 // limit inflight 10s
+
+                check { result, exception, startTime, endTime ->
+                    log.info("Stream load result: ${result}")
+                    def json = parseJson(result)
+                    assertEquals("fail", json.Status.toLowerCase())
+                    assertTrue(json.Message.contains("update delete bitmap failed"))
+                }
+            }
+            def time_cost = System.currentTimeMillis() - now
+            getTotalRetry.call()
+            assertEquals(last_total_retry + 2, total_retry)
+            qt_sql5 """ select * from ${tableName} order by id"""
+
+            // 3.2 second load will success because of removing timeout simulation
+            GetDebugPoint().disableDebugPointForAllBEs("CloudMetaMgr::test_update_delete_bitmap_fail")
+            streamLoad {
+                table "${tableName}"
+
+                set 'column_separator', ','
+                set 'columns', 'id, name, score'
+                file "test_stream_load3.csv"
+
+                time 10000 // limit inflight 10s
+
+                check { result, exception, startTime, endTime ->
+                    log.info("Stream load result: ${result}")
+                    def json = parseJson(result)
+                    assertEquals("success", json.Status.toLowerCase())
+                }
+            }
+            getTotalRetry.call()
+            assertEquals(last_total_retry + 2, total_retry)
+            qt_sql6 """ select * from ${tableName} order by id"""
+        }
+
+        //4. test wait fe lock timeout, will retry
+        setFeConfigTemporary(customFeConfig1) {
+            get_be_param("txn_commit_rpc_timeout_ms")
+            set_be_param("txn_commit_rpc_timeout_ms", "10000")
+            GetDebugPoint().enableDebugPointForAllFEs("CloudGlobalTransactionMgr.tryCommitLock.timeout", [sleep_time: 5])
+            // 4.1 first load will fail, because of waiting for fe lock timeout
+            def now = System.currentTimeMillis()
+            streamLoad {
+                table "${tableName}"
+
+                set 'column_separator', ','
+                set 'columns', 'id, name, score'
+                file "test_stream_load4.csv"
+
+                time 10000 // limit inflight 10s
+
+                check { result, exception, startTime, endTime ->
+                    log.info("Stream load result: ${result}")
+                    def json = parseJson(result)
+                    assertEquals("fail", json.Status.toLowerCase())
+                    assertTrue(json.Message.contains("get table cloud commit lock timeout"))
+                }
+            }
+            def time_cost = System.currentTimeMillis() - now
+            getTotalRetry.call()
+            assertEquals(last_total_retry + 4, total_retry)
+            assertTrue(time_cost > 10000, "wait time should bigger than total retry interval")
```

Comment:
Also add an assertion to check that the `time_cost` should be **almost** equal to the `txn_commit_rpc_timeout_ms`.

Reply:
same problem as 3.2, close